R/pkg/R/DataFrame.R

```diff
@@ -2121,7 +2121,7 @@ setMethod("arrange",
           })
 
 #' @rdname arrange
-#' @name orderBy
```

Comment:
The top of the arrange.Rd looks like

\docType{methods}
\name{arrange}
\alias{arrange}
\alias{arrange,SparkDataFrame,Column-method}
\alias{arrange,SparkDataFrame,character-method}
\alias{orderBy,SparkDataFrame,characterOrColumn-method}

orderBy.Rd looks like

\docType{methods}
\name{orderBy}
\alias{orderBy}
\alias{orderBy,WindowSpec,Column-method}
\alias{orderBy,WindowSpec,character-method}
\title{Ordering Columns in a WindowSpec}